templates/_macros/vf_cta-section.jinja

```diff
@@ -0,0 +1,105 @@
+
+    # Slots:
+        # title: When specified, the "title_text" parameter will be ignored - Optional
+
+  # cta-default-25-75: 25-75 percent width CTA with title and link text
+    # Params:
+        # title_text: H2 title text - optional
+        # link_text: H2 link text - required
+        # link_href: href for the h2 link - required
+        # layout: the variant - cta-default-25-75
+        # has_chevron: Whether the link text should have a chevron icon - Optional, default is True
+
+      # Slots:
+        # title: When specified, the "title_text" parameter will be ignored - Optional
+
+  # cta-block-100: Full width CTA with title, description and cta block with action links
+    # Params:
+        # title_text: H2 title text - required
+        # layout: the variant - cta-block-100
+
+    # Slots:
+        # description: Paragraph-style (one or more) content below the title - Optional
+        # cta: Call-to-action block (required)
+
+  # cta-block-25-75: 25-75 percent width CTA with title, description and cta block with action links
+    # Params:
+        # title_text: H2 title text - required
+        # layout: the variant - cta-block-25-75
+
+    # Slots:
+        # description: Paragraph-style (one or more) content below the title - Optional
+        # cta: Call-to-action block (required)
+
+{% macro vf_cta_section(title_text, link_text, link_href, has_chevron=True, layout='cta-default-100', caller=None) %}
+  {% set title_content = caller('title') %}
+  {% set has_title_slot = title_content|trim|length > 0 %}
+  {% set description_content = caller('description') %}
+  {% set has_description = description_content|trim|length > 0 %}
+  {% set cta_content = caller('cta') %}
+  {% set has_cta = cta_content|trim|length > 0 %}
+  {% macro _description_block() -%}
+    {% if has_description %}{{ description_content }}{% endif %}
+  {% endmacro %}
+  {% macro _cta_link() %}
+    {% if link_text and link_href %}
+      <a href="{{ link_href }}">{{ link_text }}
+        {% if has_chevron %}&#32;&rsaquo;{% endif %}
+      </a>
+    {% endif %}
+  {% endmacro %}
+  {% macro _cta_block() -%}
+    {% if has_cta -%}<div class="p-cta-block">{{ cta_content }}</div>{% endif %}
+  {% endmacro %}
+  {% macro _cta_default_variant() %}
+    <h2>
+      {% if has_title_slot %}
+        {{ title_content }}
+      {% elif title_text %}
+        {{ title_text }}
+        <br />
+        {{ _cta_link() }}
+      {% endif %}
+    </h2>
+  {% endmacro %}
+  {% macro _cta_block_variant() %}
+    <h2>{{ title_text }}</h2>
+    {{ _description_block() -}}
+    {{ _cta_block() }}
+  {% endmacro %}
+  <hr class="p-rule is-fixed-width u-no-margin--bottom" />
+  <section class="p-strip is-deep">
+    {% if layout == 'cta-default-100' %}<div class="u-fixed-width">{{ _cta_default_variant() }}</div>{% endif %}
+    {% if layout == 'cta-default-25-75' %}
+      <div class="row">
+        <div class="col-start-large-4 col-9">{{ _cta_default_variant() }}</div>
```

Comment:
Can we use the [new `.grid-row` and `.grid-col`](https://vanillaframework.io/docs/patterns/grid), here and elsewhere? This should be doable as long as there is no requirement to use col counts that are not divisible by 4.